FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Model/OAuth2/AccessTokenTests.swift

```diff
@@ -109,10 +109,11 @@ class AccessTokenTests: FRAuthBaseTest {
         XCTAssertNil(at)
         
         // Given missing scope
-        tokenDict = ["access_token":self.access_token, "token_type":self.token_type, "expires_in":self.expires_in]
-        at = AccessToken(tokenResponse: tokenDict)
-        // Then
-        XCTAssertNil(at)
+        // Removing the validation for scope as it is no longer mandatory
+//        tokenDict = ["access_token":self.access_token, "token_type":self.token_type, "expires_in":self.expires_in]
+//        at = AccessToken(tokenResponse: tokenDict)
+//        // Then
+//        XCTAssertNil(at)
```

Comment:
Probably better to leave the test and just change the assertion to `XCTAssertNotNil(at)`?

Reply:
NM... this is covered in the `Given minimal set of response values` case...